### ISSUE TITLE
the first page Consent Form - typo and wording.

### ISSUE THREAD

**Issue by tnsgmddb** [expfactory/expfactory-experiments#227]
![1_consent-typos](https://cloud.githubusercontent.com/assets/16784562/13055283/7d1e000e-d3dc-11e5-86c9-e92c09a6f51c.png)
A typo in the first sentence. 

-----------------------------------------------------wording-----

"You will be paid based on the predicted time needed for the tasks completed based on the following schedule: $1/30 min + $2 for each hour of tasks completed + any performance bonuses.

For example, your base pay would be $5 for 1.5 hours of tasks and $8 for two hours of tasks."

can be hard to follow for people with no idea how this is computed. So you may replace the second sentence ->
"For example, your base pay would be $5 for 1.5 hours of tasks {($1 x 3) + $2} ;
and $8 for two hours of tasks {($1 x 4) + ($2 x 2)} ."

**Comment by IanEisenberg** [comment]
Made changes as suggested.
